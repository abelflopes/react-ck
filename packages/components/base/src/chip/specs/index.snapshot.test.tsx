import React from "react";
import { Chip } from "../index";
import renderer from "react-test-renderer";

describe("snapshot Chip", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Chip>Chip</Chip>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
