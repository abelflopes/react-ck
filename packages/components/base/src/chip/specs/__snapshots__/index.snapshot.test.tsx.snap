// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`snapshot Chip renders correctly 1`] = `
<DocumentFragment>
  <span
    class="root skin_neutral variation_filled size_m interaction_undefined"
  >
    Chip
  </span>
</DocumentFragment>
`;
